var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -22,10 +22,10 @@ class PyNeurodamus(PythonPackage):
     depends_on('py-setuptools',    type=('build', 'run'))
     depends_on('py-h5py',          type='run')
     depends_on('py-numpy',         type='run')
-    depends_on('py-lazy-property', type='run')
     depends_on('py-docopt',        type='run')
-    depends_on('py-six',           type='run')
-    depends_on('py-scipy',         type='run')
+    depends_on('py-lazy-property', type='run', when='@:1.0.0')
+    depends_on('py-six',           type='run', when='@:1.0.0')
+    depends_on('py-scipy',         type='run', when='@1.2.0:')
```

Comment:
```suggestion
    depends_on('py-scipy',         type='run', when='@1.1.0:')
```
Should this be `1.1.0:`?

Reply:
https://bbpcode.epfl.ch/code/#/c/49031/ is not merged yet and I think it will make a new minor bump. But true, it's not usable like this. Maybe we can use @develop ? that will slow down tests, but well...

Reply:
actually, as is, @develop is probably included, or not?

Reply:
Ah I see. You're looking at the future 😛